### PR TITLE
Welcome e2e tests

### PR DESCRIPTION
**Pull request by boodland**
### Issue link / number:
#1089

### What changes did you make?
Created welcome and welcome badoo e2e tests for english version

### Why did you make the changes?
So the page is covered with test and help us to validate it is working when making changes or migrating it to app router

### Did you run tests?
Yes

### CODE REVIEW

**Comment by boodland** [comment]
@kyleecodes a follow up issue/ticket can be created to add tests for the rest of languages and partners (bumble and fruitz), I did not include all of them to save time and be able to move to another migration task, also they can be good first tasks for new contributors. Thanks in advance

**Review by eleanorreem — DISMISSED** [review]

File: cypress/integration/welcome.badoo.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      cy.get(labelAccessCodeTag).should('exist').should('have.attr', 'for', 'accessCode');
+      cy.get(inputAccessCodeTag).should('exist');
+    });
+    describe('navigate to register page when clicking get started action', () => {
+      it('without action', () => {
+        const registerPartnerBadooUrl = '/auth/register?partner=badoo';
+        cy.get('button').contains('Get started').click();
+        cy.checkPageUrl(registerPartnerBadooUrl);
+      });
+      it('with action code', () => {
```
Comment (eleanorreem) [inline cypress/integration/welcome.badoo.cy.tsx:81]: So i'd drop this test `with action code`. What you have done, tests enough. We also have other tests that loosely follow the same flow so this doesn't need to be thorough. See register-partner-with-code.tsx and register-partner-without-code.tsx. Also you might have a different setup as this didn't work for me.

**Comment by eleanorreem** [comment]
@boodland thanks for this! Nice to safeguard your changes. I think you might be duplicating some cypress tests so I'd keep this test a bit simpler. See create-user.tsx, register-partner-with-code.tsx and register-partner-without-code.tsx.

File: cypress/integration/welcome.badoo.cy.tsx (hunk 1 above)
Comment (boodland) [reply cypress/integration/welcome.badoo.cy.tsx]: @eleanorreem I have applied the changes.Thanks for the feedback!

**Comment by eleanorreem** [comment]
Thanks @boodland

**Pull request merged**